README.md

```diff
@@ -49,8 +49,9 @@ You need:
 
 Example:
 ```bash
-cmake . -DLLVM_CONFIG_EXECUTABLE=/opt/llvm/bin/llvm-config -DCMAKE_BUILD_TYPE=Release
-make
+mkdir -p build && cd build
+cmake .. -DLLVM_CONFIG_EXECUTABLE=llvm-config-3.8 -DCMAKE_INSTALL_PREFIX=$HOME/woboq.install -DCMAKE_BUILD_TYPE=Release
```

Comment:
But the README does not tell to create a build directory.

I personaly use shadow build., but i think in-source builod are simpler to explain.  And anybody familiar with cmake knows how to make a shadow build.